framework/src/preconditioners/MoosePreconditioner.C

```diff
@@ -27,7 +27,7 @@ InputParameters validParams<MoosePreconditioner>()
   InputParameters params = validParams<MooseObject>();
   params.addPrivateParam<FEProblemBase *>("_fe_problem_base");
 
-  MooseEnum pc_side("left right symmetric", "right");
+  MooseEnum pc_side("left right symmetric");
```

Comment:
What do you think about adding `default` value which would be the default ;-) And if it set, we do not change the preconditioner side...